schemas/src/registry_schemas/schemas/name_request.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request",
+    "type": "object",
+    "title": "Name Request Schema",
+    "properties": {
+        "nrNumber": {
+            "type": "string"
+        },
+        "nrName": {
+            "type": "string"
+        },
+        "legalType": {
+            "type": "string",
+            "title": "The Legal Type of the business.",
+            "default": "BC",
+            "enum": [
+                "BC",
+                "CP"
+            ]
+        },
+        "requestType": {
```

Comment:
Can you add some description for some of these fields? I'm unclear how this differs from legalType.

Reply:
Absolutely, but to be honest i don't really know what it means :laugh. I can certainly take a crack at giving them a better definition though :).         
                                                               
I just pulled these properties off of the design. For this specific property, the value given ( as per design mockup ) was `New Business`.  Attached is a link to the mock up.
 
https://preview.uxpin.com/523b0c07ed451c8c3a417a9b758e194424fc1ead#/pages/121388482/simulate/sitemap